sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ThroughputControlGroup.java

```diff
@@ -0,0 +1,212 @@
+    public CosmosAsyncContainer getTargetContainer() {
+        return this.targetContainer;
+    }
+
+    /**
+     * Get throughput control group target throughput.
+     * Since we allow either TargetThroughput or TargetThroughputThreshold, this value can be null.
+     *
+     * By default, it is null.
+     *
+     * @return the target throughput.
+     */
+    @Beta(value = Beta.SinceVersion.WHATEVER_NEW_VERSION, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public Integer getTargetThroughput() {
+        return this.targetThroughput;
+    }
+
+    /**
+     * Set the throughput control group target throughput. The value should be larger than 0.
+     *
+     * @param targetThroughput the target throughput for the throughput control group.
+     * @return the {@link ThroughputControlGroup}.
+     */
+    @Beta(value = Beta.SinceVersion.WHATEVER_NEW_VERSION, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public ThroughputControlGroup setTargetThroughput(int targetThroughput) {
+        checkArgument(targetThroughput > 0, "Target throughput should be larger than 0");
+        this.targetThroughput = targetThroughput;
+        return this;
+    }
+
+    /**
+     * Get the throughput control group target throughput threshold.
+     * Since we allow either TargetThroughput or TargetThroughputThreshold, this value can be null.
+     *
+     * By default, this value is null.
+     *
+     * @return the target throughput threshold.
+     */
+    @Beta(value = Beta.SinceVersion.WHATEVER_NEW_VERSION, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public Double getTargetThroughputThreshold() {
+        return this.targetThroughputThreshold;
+    }
+
+    /**
+     * Set throughput control group target throughput threshold. The value should be larger than 0.
+     *
+     * @param targetThroughputThreshold the target throughput threshold for the throughput control group.
+     *
+     * @return the {@link ThroughputControlGroup}.
+     */
+    @Beta(value = Beta.SinceVersion.WHATEVER_NEW_VERSION, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public ThroughputControlGroup setTargetThroughputThreshold(double targetThroughputThreshold) {
+        checkArgument(
+            targetThroughputThreshold > 0 && targetThroughputThreshold <= 1,
+            "Target throughput threshold should between (0, 1]");
+
+        this.targetThroughputThreshold = targetThroughputThreshold;
+        return this;
+    }
+
+    /**
+     * Get whether this throughput control group will be used by default.
+     *
+     * By default, it is false.
+     *
+     * @return {@code true} this throughput control group will be used by default unless being override. {@code false} otherwise.
+     */
+    @Beta(value = Beta.SinceVersion.WHATEVER_NEW_VERSION, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public boolean isUseByDefault() {
```

Comment:
Chat offline, for now will keep isUseByDefault

Reply:
isUsedByDefault -> isDefault
No need for extra wording in the method name, it's clear that if default then that's what it will be used unless other is specified.